kubernetes/services/rabbitmq.yml.j2

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rabbitmq
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: rabbitmq
+    spec:
+      hostname: rabbitmq-1
+      containers:
+        - image:           rabbitmq:3.7.4
+          name:            rabbitmq
+          imagePullPolicy: Always
+          env:
+            - name:  RABBITMQ_MNESIA_BASE
+              value: /srv/storage/mnesia
+          resources:
+            requests:
+              memory: "1200Mi"
+              cpu:    "600m"
+            limits:
+              memory: "1600Mi"
+              cpu:    "900m"
+          volumeMounts:
+            - mountPath:  /srv/storage/mnesia/
+              name:       queue-storage
+          livenessProbe:
+            exec:
+              command:
+              - rabbitmqctl
+              - status
+            initialDelaySeconds: 120
+            timeoutSeconds: 5
+            failureThreshold: 6
```

Comment:
How were these values established?

Also, please align this.

Reply:
Done. Values are from `kubernetes` github repository: https://github.com/kubernetes/charts/blob/master/stable/rabbitmq/values.yaml

Reply:
OK. It's fine as a starting point then.

In general, please try to use more comments explaining your reasoning in the configs.